### DISCUSSION TITLE
Information on Neuromorphic course/rpository

### DISCUSSION THREAD

**Discussion by crsumiran** [HerrNamenlos123/JTAG_Interface#9] [General]
Hi,
@HerrNamenlos123, I have collaborated with a team that worked on a neuromorphic accelerator on the application side but not on the hardware and would like to know more about what you guys implemented and experimented with. 
As you mentioned that you have used the board to implement projects on neuromorphic computing, could you please share the link to that repository. 

Regards
Ranga

**Comment by HerrNamenlos123** [comment]
Hi @crsumiran, thank you for reaching out. Unfortunately I cannot give you any information as I have never worked on any projects related to neuromorphic computing, nor have I access to them. I never claimed to have ever been part of any projects.

A professor from the University Nijmegen reached out to me because he and his students at University perform neuromorphic computing using my library and they had an issue which I later fixed. I have no clue what they are doing. As such, all I have is an EMail address of the Professor responsible for the course, and his permission to talk about it in my portfolio.
